src/main/slice/omero/cmd/FS.ice

```diff
@@ -76,7 +76,7 @@ module omero {
          * will be returned under normal conditions, otherwise a {@link ERR}
          * will be returned.
          **/
-        class OriginalMetadataRequest extends Request {
+        class OriginalMetadataRequest extends OK {
```

Comment:
This should be reverted. Training failed https://web-proxy.openmicroscopy.org/west-ci/job/OMERO-training/335